src/qwed_new/core/cache.py

```
@@ -162,312 +224,467 @@ def stats(self) -> Dict[str, Any]:
                 "backend": "redis",
+                "mode": self.mode,
+                "degraded": False,
                 "hits": self._hits,
                 "misses": self._misses,
                 "hit_rate_percent": round(hit_rate, 2),
                 "redis_used_memory": info.get("used_memory_human"),
                 "enabled": self.enabled,
-                "tenant_id": self.tenant_id
+                "tenant_id": self.tenant_id,
             }
         except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)